### PR TITLE
14.0.0.DEV03 Indexing features blog post

### PR DESCRIPTION
**Pull request by fax4ever**

### CODE REVIEW

**Comment by fax4ever** [comment]
With the last change, I just updated my picture. Maybe it looks more professional. I don't know :P

**Review by karesti — APPROVED** [review]

**Comment by karesti** [comment]
LGTM @fax4ever ! Anyone else that might want to review de english aspects of the post maybe ? @tristantarrant @dvagnero

File: _posts/2022-05-30-infinispan-14-indexing-query-news.adoc [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+Embedded indexes are defined using the `@Embedded` annotation, and it is possible to choose between the `NESTED` structure which
+preserves the original object relationship structure and `FLATTENED` structure which makes the leaf fields multi-valued
+fields of the parent entity.
+
+== Index startup mode
+
+Sometimes indexes can be persistent and cache data not or vice versa, for those cases it can be useful
+to perform some operations to ensure the index will be consistent with data in the cache.
+
+We introduced the `startup-mode` configuration. Here is an example:
```
[the remark this answers is not in the document]
Comment (fax4ever) [reply _posts/2022-05-30-infinispan-14-indexing-query-news.adoc:92]: Yes we didn't do that, because the `startup-mode` is a Dev03 feature.

**Comment by fax4ever** [comment]
Thanks, @karesti!

**Comment by tristantarrant** [comment]
Let's publish this

**Comment by fax4ever** [comment]
Merged, thank!

**Comment by dvagnero** [comment]
@fax4ever I had a review in progress but since it's merged we can leave it as it is.

**Comment by fax4ever** [comment]
> @fax4ever I had a review in progress but since it's merged we can leave it as it is.

I'm sorry @dvagnero :(

**Pull request merged**